codebuild/create_project.py

```diff
@@ -18,16 +18,70 @@
 import configparser
 import logging
 
-from awacs.aws import Allow, Statement, Principal, PolicyDocument
+from awacs.aws import Action, Allow, Statement, Principal, PolicyDocument
 from awacs.sts import AssumeRole
-from troposphere import Template, Ref, Output
+from random import randrange
+from troposphere import GetAtt, Template, Ref, Output
+from troposphere.events import Rule, Target
 from troposphere.iam import Role
 from troposphere.codebuild import Artifacts, Environment, Source, Project
 
 logging.getLogger(__name__)
 logging.basicConfig(level=logging.DEBUG)
 
 
+def build_cw_event(template=Template, project_name=None, role=None):
+    # Run during the business day PDT
+    hour = randrange(14, 23)
```

Comment:
Why random? This updates every time the configuration is changed?

Reply:
The goal was to avoid blocking in-flight PR testing.  Another review asked we move it to after business hours PST, which we're considering; as long as we all agree breaking master builds are not pageable.